### PR TITLE
Kubernetes - FINAL

### PR DESCRIPTION
**Pull request by brenodt**
Issue:
 
# Relevant motivation and context

# Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
 
# Checklist
- Have you set a PR title that is short and concise?
- Have you used labels like `wip`, `review`, `backend`, `unity`, etc...?
- Have you performed a self-review of your own code?
- Have you commented your code, particularly in hard-to-understand areas?
- Have you made corresponding changes to the documentation?
- Have you added tests that prove your fixes are effective and work as expected?
- Have you ran new and existing tests and they pass locally with my changes?

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,17 +1,53 @@
 version: 2.1
 orbs:
   slack: circleci/slack@3.4.2
+  kubernetes: circleci/kubernetes@0.11.0
+  digitalocean: digitalocean/cli@0.1.1
+  docker: circleci/docker@1.0.1
+  envsubst: sawadashota/envsubst@1.1.0
+
```
Comment (diegoseso) [inline .circleci/config.yml:8]: remove empty line

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,17 +1,53 @@
...
 workflows:
   build:
     jobs:
       - build-go-proto-files
       - build-dart-proto-files
+
```
Comment (diegoseso) [inline .circleci/config.yml:14]: remove comment and empty line

File: .circleci/config.yml [hunk 3]
```diff
@@ -1,17 +1,53 @@
...
+          requires:
+            - update-container-registry
+          filters:
+            branches:
+              only: master
+          context: digital-ocean
+          container: registry.digitalocean.com/takumi-software/blisser-backend-api
+          cluster-name: k8s-takumi-software-001
+          tag: "latest"
+
```
Comment (diegoseso) [inline .circleci/config.yml:49]: remove empty line, comment

File: deploy/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,9 @@
+FROM alpine:latest
+
+COPY blisser-backend-api /usr/local/bin/blisser-backend-api
+COPY backend-server.yml /usr/local/bin/backend-server.yml
+WORKDIR /usr/local/bin/
+ENTRYPOINT ./blisser-backend-api
+
+EXPOSE 8002
+EXPOSE 8000
```
Comment (diegoseso) [inline deploy/Dockerfile:9]: can you expose it through the 80 port please?

File: deploy/scripts/ci-deploy.sh [hunk 5]
```diff
@@ -0,0 +1,23 @@
+#! /bin/bash
+# exit script when any command ran here returns with non-zero exit code
+set -e
+
+COMMIT_SHA1=$CIRCLE_SHA1
+
+# We must export it so it's available for envsubst
+export COMMIT_SHA1=$COMMIT_SHA1
+
+# since the only way for envsubst to work on files is using input/output redirection,
```
Comment (diegoseso) [inline deploy/scripts/ci-deploy.sh:10]: remove the comment please

File: deploy/scripts/ci-deploy.sh [hunk 6]
```diff
@@ -0,0 +1,23 @@
+#! /bin/bash
+# exit script when any command ran here returns with non-zero exit code
+set -e
+
+COMMIT_SHA1=$CIRCLE_SHA1
+
+# We must export it so it's available for envsubst
```
Comment (diegoseso) [inline deploy/scripts/ci-deploy.sh:7]: remove comment please

Comment (brenodt) [reply deploy/scripts/ci-deploy.sh]: I removed the file, instead. We're not using it for deployment. Makes more sense

**Review by brenodt — COMMENTED** [review]
No further issues.

**Pull request merged**